src/main/java/fr/techad/edc/popover/utils/OpenUrlAction.java

```diff
@@ -38,23 +114,19 @@ public OpenUrlAction(Browser browser, HelpConfiguration helpConfiguration) {
      * @throws IOException        if the connexion failed
      * @throws URISyntaxException the url is malformed
      */
-    public void openUrl(String url) throws IOException, URISyntaxException {
+    public void openUrl(String url) throws Exception {
         LOGGER.debug("Open the url: {}", url);
 
         if(helpConfiguration.getHelpViewer() == HelpViewer.EDC_DESKTOP_VIEWER){
-            Runtime runtime = Runtime.getRuntime();
-            try
-            {
-                if(helpConfiguration.getViewerDesktopPath().isEmpty()){
-                    LOGGER.error("The path of the application must be entered");
-                }else{
-                    runtime.exec(helpConfiguration.getViewerDesktopPath());
-                    LOGGER.info("Desktop viewer is running");
-                }
-            }
-            catch (IOException e)
-            {
-                LOGGER.error(String.valueOf(e));
+            if(helpConfiguration.getViewerDesktopPath().isEmpty()){
+                LOGGER.error("The path of the application must be entered");
+            }else {
+                isAlive();
+
+                StreamGobbler streamGobbler = new StreamGobbler(edcDesktopProcess.getInputStream(), System.out::println);
+                Executors.newSingleThreadExecutor().submit(streamGobbler);
+
+                setInterval(url);
```

Comment:
setInterval and you define an url.

Are you sure that the method name is correct?